geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/DumpQueriesCommand.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2014 Commonwealth Computer Research, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.locationtech.geomesa.tools.commands
+
+import java.io._
+import java.util.Date
+
+import au.com.bytecode.opencsv.CSVWriter
+import com.beust.jcommander.{JCommander, Parameter, Parameters}
+import org.locationtech.geomesa.core.stats.RasterQueryStatTransform
+import org.locationtech.geomesa.raster.data.AccumuloCoverageStore
+import org.locationtech.geomesa.tools.AccumuloProperties
+import org.locationtech.geomesa.tools.commands.DumpQueriesCommand.{Command, DumpQueriesParameters}
+
+import scala.collection.JavaConverters._
+
+class DumpQueriesCommand(parent: JCommander) extends Command with AccumuloProperties {
+
+  val params = new DumpQueriesParameters()
+  parent.addCommand(Command, params)
+
+  override def execute() = {
+    val queryRecords = createCoverageStore(params).getQueryRecords(params.numRecords * RasterQueryStatTransform.NUMBER_OF_CQ_DATA_TYPES)
+    val fw = getFileWriter
+    val out = new BufferedWriter(fw)
+    val writer = new CSVWriter(out)
+    writer.writeAll(queryRecords.asJava)
```

Comment:
I think we should change getQueryRecords to return an Iterable/Iterator.  If the user asked for a large number of records, the .toList would materialize them all in the memory...